src/game/game/game.c

```diff
@@ -203,10 +204,6 @@ GAMEFLOW_OPTION Game_Stop(void)
         return GF_LEVEL_COMPLETE | g_GameInfo.select_level_num;
     }
 
-    if (!g_InvChosen) {
-        return GF_EXIT_TO_TITLE;
-    }
```

Comment:
@lahm86 from what I could tell, this is never triggered. I tried quitting to title from anywhere, final stats, level skipping in the inventory, etc. I think it can be removed but I'm not sure. This is also related to `phase_inventory.c`'s `Inv_Construct` which is the only place where `g_InvChosen = 0;` was set. Because of the change to `GAME_OBJECT_ID`, setting it to `0` was no longer proper. But, idk if there is some special edge case where it's relevant.

Reply:
As far as I can tell, this looks fine to be removed. I can't think of any special case where it would be needed here.